apis/sprints.py

```diff
@@ -147,9 +144,6 @@ def get_prompt_leaderboard(id, run_id):
     user_id = session.get("user_id")
     prompt = prompts.get_prompt(id, "sprint", user_id=user_id)
 
-    if not session.get("admin", False) and (prompt["active"] and prompt["rated"] and not prompt.get("played", False)):
```

Comment:
I think this would be related to the spoilers things we wanted to implement?

Reply:
Hmm I think it feels better if it's still hidden until you play it. We'll have the spoilers thing too but for right now we don't. Even when we do have it, I think it still makes sense like this

Reply:
Then for anonymous users, we'll have to check locally stored runs in order to determine whether they've attempted it or not, which will be done on the front end

Reply:
Added. So now when attempting to load the leaderboard, frontend will first check if the user is not logged in AND hasn't done the prompt yet. If so, error out to home page. 

However, frontend does not know if the leaderboard accessed is the active potd, so it will just shun out anony users for any prompt if they haven't attempted yet. This will need to be fixed